packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -0,0 +1,511 @@
+    const eltChildren: JSX.Element[] = [];
+
+    let codeIndex = 0;
+    let textIndex = 0;
+    while (textIndex < text.length && codeIndex < codeBlocks.length) {
+      const codeBlock = codeBlocks[codeIndex];
+      if (textIndex < codeBlock.index) {
+        const str = text.substring(textIndex, codeBlock.index);
+        eltChildren.push(<span key={textIndex}>{str}</span>);
+        textIndex += str.length;
+      } else {
+        eltChildren.push(<code key={textIndex}>{codeBlock.text.substring(1, codeBlock.text.length - 1)}</code>);
+        codeIndex++;
+        textIndex += codeBlock.text.length;
+      }
+    }
+    if (textIndex < text.length) {
+      eltChildren.push(<span key={textIndex}>{text.substring(textIndex, text.length)}</span>);
+    }
+
+    return eltChildren;
+  }
+
+  private _renderClass(): JSX.Element | undefined {
+    const { properties, methods } = this.state;
+
+    return methods ? (
+      <Stack gap={MEDIUM_GAP_SIZE}>
+        <Stack gap={SMALL_GAP_SIZE}>
+          <Text variant={'medium'}>Members</Text>
+          <DetailsList
+            selectionMode={SelectionMode.none}
+            layoutMode={DetailsListLayoutMode.justified}
+            items={properties}
+            columns={this._defaultColumns}
+            onRenderRow={this._onRenderRow}
+          />
+        </Stack>
+        <Stack gap={SMALL_GAP_SIZE}>
+          <Text variant={'medium'}>Methods</Text>
+          <DetailsList
+            selectionMode={SelectionMode.none}
+            layoutMode={DetailsListLayoutMode.justified}
+            items={methods}
+            columns={this._methodColumns}
+            onRenderRow={this._onRenderRow}
+          />
+        </Stack>
+      </Stack>
+    ) : (
+      <Stack gap={SMALL_GAP_SIZE}>
+        <Text variant={'medium'}>Members</Text>
+        <DetailsList
+          selectionMode={SelectionMode.none}
+          layoutMode={DetailsListLayoutMode.justified}
+          items={properties}
+          columns={this._defaultColumns}
+          onRenderRow={this._onRenderRow}
+        />
+      </Stack>
+    );
+  }
+
+  private _renderExtends(): JSX.Element | undefined {
+    const { extendsTokens } = this.props;
+
+    return this._parseILinkTokens(true, extendsTokens);
+  }
+
+  private _renderDescription(): JSX.Element | undefined {
+    const { description } = this.props;
+
+    return description ? <Text variant={'medium'}>{description}</Text> : undefined;
+  }
+
+  private _renderTitle(): JSX.Element | undefined {
+    const { title, name } = this.props;
+
+    return title ? (
+      <Text variant={'xLarge'} id={name}>
+        {title}
+      </Text>
+    ) : (
+      undefined
+    );
+  }
+
+  private _onRenderRow = (props: IDetailsRowProps, defaultRender?: IRenderFunction<IDetailsRowProps>): JSX.Element => {
+    const { item } = props;
+    const rowStyles: Partial<IDetailsRowStyles> = {
+      root: {
+        selectors: {
+          ':hover': {
+            background: 'transparent'
```

Comment:
```suggestion
            background: 'none'
```

Curious, does the above not work?

Reply:
Nope, it definitely works, just honestly didn't think to do that!